packages/html2react/processors/__tests__/link.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { createStore } from "@frontity/connect";
+import { Html2React } from "../../src/libraries/component";
+import link from "../link";
+import Html2ReactPackage from "../../src";
+
+const store = createStore(Html2ReactPackage);
+store.libraries.html2react.processors.push(link);
+
+jest.mock("@frontity/components/link", () => "mocked-link");
+
+describe("Link Processor", () => {
+  it("should replace all anchor tags with links", () => {
+    const { container } = render(
+      <Html2React
+        html="<a href='http://my-link.com/post-name'>My Link</a><a href='http://my-link.com/post-name'>My Link 2</a>"
+        {...store}
```

Comment:
@DAreRodz maybe we can simplify the processor tests now that you have added a `processors` prop to `Html2React`.

What do you think? Would something like this work?

```js
<Html2React
  html="<a href='http://my...ame'>My Link 2</a>"
  processors={[link]}
/>
```

Reply:
Sure, that should work fine. When the `processors` prop is present, the content of `libraries.html2react.processors` is just ignored.

Reply:
FYI: TS complains about types not matching if we just pass processors since the other props aren't optional.

Reply:
Oh, I see. That happens because the _disconnected_ `Html2React` is being imported

```ts
import { Html2React } from "../../src/libraries/component";
```
instead of the _connected_ version, which is the default export:

```ts
import Html2React from "../../src/libraries/component";
```

That one should not require the Frontity store props to be passed. :slightly_smiling_face:

Reply:
That worked, updated 👍